spec/traits.dd

```diff
@@ -1020,6 +1020,59 @@ void main()
         $(P If the two arguments are expressions made up of literals
         or enums that evaluate to the same value, true is returned.)
 
+        $(P If the two arguments are both lambda functions (or aliases
+        to lambda functions), then they are compared for equality. For
+        the comparison to be computed correctly, the following conditions
+        must be met for both lambda functions:)
+
+        $(OL
+        $(LI The lambda function arguments must not have a template
+        instantiation as an explicit argument type. Any other argument
+        types (basic, user-defined, template) are supported.)
+        $(LI the lambda function body must contain a single expression
+        (no return statement) which contains only numeric values,
+        manifest constants, enum values and function arguments. If the
+        expression contains local variables, function calls or return
+        statements, the function is considered uncomparable.)
+        )
+
+---
```

Comment:
Try the new `SPEC_RUNNALE_EXAMPLE_COMPILE` macro
Verify with `pmake test_dspec`

Example: https://github.com/dlang/dlang.org/blob/47bfef14e4275f42b403d75b31628d563f57b1d4/spec/type.dd